proxy/src/main/java/ru/leymooo/botfilter/BotFilter.java

```diff
@@ -74,9 +74,13 @@ public BotFilter(boolean startup)
         Settings.IMP.reload( new File( "BotFilter", "config.yml" ) );
         Scoreboard.DISABLE_DUBLICATE = Settings.IMP.FIX_SCOREBOARD_TEAMS;
         checkForUpdates( startup );
-        if ( !CachedCaptcha.generated )
+        //Глушим исключение при попытке сгенерировать капчу. Если капча уже генерируется и бот фильтр был почему-то
+        //перезапущен, капча сгенерируется все равно и доступ к ней будет прежний.
+        try
         {
             CaptchaGeneration.generateImages();
+        } catch ( CaptchaGenerationException ignored )
```

Comment:
Don't just ignore Exceptions. Either provide a config option for that or print a message every time

Reply:
Alright